auth/src/main/res/layout-land/auth_method_picker_layout.xml

```diff
@@ -1,42 +1,59 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
```

Comment:
Try to make the changes simpler - don't remove xmlns:tools, keep the same order of parameters. No refactoring please.